index.html

```diff
@@ -1288,6 +1286,55 @@ <h2>DID Subject</h2>
 property. The value of <code>id</code> in the resolved <a>DID document</a> is
 expected to match the <a>DID</a> that was resolved.
       </p>
+        </section>
+
+        <section>
+        <h2>alsoKnownAs</h2>
+        <p>
+        A <a>DID controller</a> MAY assert that two or more <a>DIDs</a> identify
+        the same <a>DID subject</a> using the <code>alsoKnownAs</code> property.
+        This property is OPTIONAL.
+        </p>
+
+        <dl>
+        <dt>alsoKnownAs</dt>
+        <dd>
+        The value of <code>alsoKnownAs</code> MUST be a
+        <a data-cite="INFRA#lists">list</a> where each item in the list is a
+        <a>DID</a> that conforms to the rules in Section <a href="#did-syntax"></a>.
+        </dd>
+      </dl>
+        <p>
+        From a semantic standpoint, the <a>DID controller</a> is asserting that
+        each <a>DID</a> in the list identifies the same <a>DID subject</a> as
+        the <a>DID</a> described by the <a>DID document</a> (i.e., the value of
+        the <code>id</code> property).
+        </p>
+        <p class="note">
+        WARNING: An <code>alsoKnownAs</code> assertion that two or more <a>DIDs</a>
+        identify the same <a>DID subject</a> does not prove this assertion is
+        true. A malicious <a>DID controller</a> might attempt to impersonate a
+        <a>DID subject</a> by making a false <code>alsoKnownAs</code> assertion.
+        Therefore it is strongly advised that a requesting party obtain
+        independent verification of a <code>alsoKnownAs</code> assertion. One
+        method for doing this is to resolve the <code>alsoKnownAs</code> <a>DID</a>
+        to its own <a>DID document</a> and then verify that it contains the
+        inverse <code>alsoKnownAs</code> assertion, also known as a backpointer.
+        See the <a href="#security-considerations"></a> and
```

Comment:
We'll want to point to a specific item in security/privacy considerations. Just generally pointing there isn't very useful.